dist/lightbox-dialog/lightbox-dialog.css

```diff
@@ -240,6 +240,20 @@ button.icon-btn.lightbox-dialog__prev {
 [dir="rtl"] button.icon-btn.lightbox-dialog__prev .icon--chevron-left-16 {
     transform: rotate(180deg);
 }
+
+@media (max-width: 512px) {
+    .lightbox-dialog--wide .lightbox-dialog__window {
```

Comment:
I'm a bit hesitant allowing true fullscreen dialogs. I think it will perpetuate more app-like treatment possibly using stacked sheets, changing screens inside, etc. Can we use the pattern we discussed where the bit of gray appears all around with border radius? 
![image](https://github.com/eBay/skin/assets/1675667/4d8f8d29-2172-4039-833c-24d096f70e2c)

Reply:
Agreed, let's go ahead with an 8px or so area of mask around the dialog on small screens. If there is huge pushback to retain the "full" version, we can _think_ about adding it in back in for another major version or so to give apps more time to get used to the shift.

Reply:
I added a hidden `fullscreen` modifier for now. We can always take it out. 
Its not docuented